servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/BaseTestNessieApi.java

```diff
@@ -1269,7 +1280,11 @@ public void namespaces() throws Exception {
     if (isV2()) {
       soft.assertThatThrownBy(
               () -> api().deleteNamespace().refName(mainName).namespace(namespace2).delete())
-          .isInstanceOf(NessieNamespaceNotEmptyException.class);
+          .isInstanceOf(NessieNamespaceNotEmptyException.class)
+          .asInstanceOf(type(NessieNamespaceNotEmptyException.class))
```

Comment:
nit: (mostly wondering) do we need both `isInstanceOf` and `asInstanceOf`?

Reply:
think so